src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 40;
+
+  if (days >= 7) {
+    return (cost = cost * days - 50);
+  }
+
+  if (days >= 3) {
+    return (cost = cost * days - 20);
+  }
```

Comment:
50 and 20 are better be named - to make code more explanatory
please provide variables for both